BDArmory/Competition/BDACompetitionMode.cs

```diff
@@ -534,6 +535,16 @@ IEnumerator DogfightCompetitionModeRoutine(float distance, bool startDespiteFail
                                     var HPT = part.Current.FindModuleImplementing<HitpointTracker>();
                                     HPT.defenseMutator = (float)(1 / BDArmorySettings.HOS_DMG);
                                 }
+                                if (BDArmorySettings.HOS_SAS)
+                                {
+                                    if (part.Current.GetComponent<ModuleReactionWheel>() != null)
+                                    {
+                                        ModuleReactionWheel SAS;
+                                        SAS = part.Current.GetComponent<ModuleReactionWheel>();
+                                        //if (part.Current.CrewCapacity == 0)
```

Comment:
Is this meant to be commented out? The comment below indicates that SAS from cockpits is allowed, but I only vaguely recall some discussion in loggins about it.
Also below in WaypointFollowingStrategy.cs.

Reply:
Yes. The HOS version culls all RWs, the standard one a little later in the code using MAX_SAS_TORQUE ignores cockpits